src/firmware/main/math/polynomial_1d.c

```diff
@@ -1,6 +1,7 @@
 #include "firmware/main/math/polynomial_1d.h"
 
 #include <math.h>
+#include <stddef.h>
```

Comment:
@garethellis0 you mentioned in Slack you had an idea of the cause? (Seems to be the same failure/file as Haomiao). Any idea why adding the include to the header file doesn't work?

Reply:
I moved the studded.h file to the header and it works 👍